### DISCUSSION TITLE
BackgroundMusic is using my mic, according to Control Center

### DISCUSSION THREAD

**Discussion by RandomDennisLee** [kyleneideck/BackgroundMusic#653] [Q&A] [answered]
Version: 0.4.0.pkg
OS: Ventura 13.1

Whenever I run Background Music, I get an orange dot next to CC informing me that my mic is active and in use by the app. I doubt that's intended behaviour, any idea what might be causing it?

**Comment by ElectricS01** [comment]
Also happens to me, I assumed this was intended

**Reply by RandomDennisLee** [reply]
That seems very unsafe, if it is indeed intended behaviour. No app should be accessing my mic all the time, and if it isn't, it would mask notifications from other apps accessing it.

**Comment by kyleneideck** [comment, marked as answer]
I explained this a bit here: https://github.com/kyleneideck/BackgroundMusic/issues/562#issuecomment-986609439

Basically, Apple added the orange dot after we had already written Background Music to use a virtual input device, and now it would take a fair amount of work to change that. It would be worth doing, and if anyone else is interested in working on it, I'd be happy to help with that, but I doubt I'll find time for it anytime soon.
